_faqs/faq-12.md

```diff
@@ -0,0 +1,8 @@
+---
+question: I want to cite RIOT. Which reference should I use?
+---
+
+Please consider the following references for citation.
+
+  * Emmanuel Baccelli, Cenk G&uuml;ndo&#287;an, Oliver Hahm, Peter Kietzmann, Martine Lenders, Hauke Petersen, Kaspar Schleiser, Thomas C. Schmidt, Matthias W&auml;hlisch, <a href="{{ "assets/pdf/riot-ieeeiotjournal-2018.pdf" | relative_url }}">RIOT: An Open Source Operating System for Low-End Embedded Devices in the IoT</a>, IEEE Internet of Things Journal, Vol. 5, No. 6, pp. 4428-4440, December 2018.
```

Comment:
The link to the document is not correct, it should be assets/pdf**s**...

Reply:
arg. Jekyll didn't update locally. thanks!. fixed.